### ISSUE TITLE
Implement Generic Search Bar controller

### ISSUE THREAD

**Issue by daniboube** [sitmun/sitmun-viewer-app#31]
It is necessary to implement a configurable control for generic searches by street, neighborhood, municipality, cadastral reference, etc. SITNA includes one, but if it is not configured correctly, the searches will be applied to the Navarra databases.

**Comment by daniboube** [comment]
Implemented in commit a656c92bdebd878b76b1e23a0a20a9d202fe3940

The SITNA search bar is now capable of querying WFS databases served from the backend. To do this, optional parameters must be provided within the search control that match the types defined by the SITNA API for this case: [https://sitna.navarra.es/api/doc/SITNA.control.html#.SearchOptions](https://sitna.navarra.es/api/doc/SITNA.control.html#.SearchOptions)

It is important to take into account the default values ​​of these fields, since some are enabled by default, so if it is not given a new reference to search on, it will perform queries on the IDENA database.

**Comment by fjlopez** [comment]
Do you refer to this line?
https://github.com/sitmun/sitmun-viewer-app/commit/a656c92bdebd878b76b1e23a0a20a9d202fe3940#r141474226

**Comment by daniboube** [comment]
Yes, exactly that. A parameters object must be provided by the backend in the same way that is defined in the SITNA API documentation (with the optional parameters cadastralParcel, municipality, postalAddress, road, etc)

> Do you refer to this line? [a656c92#r141474226](https://github.com/sitmun/sitmun-viewer-app/commit/a656c92bdebd878b76b1e23a0a20a9d202fe3940#r141474226)

**Comment by RicardCots** [comment]
Related with [#172](https://github.com/sitmun/sitmun-backend-core/issues/172)
